1.1/spec/11-Part-08.adoc

```diff
@@ -71,6 +71,15 @@ geo:GeometryCollection
 
 The restriction imposed on the more general <<Class: SpatialObjectCollection, Spatial Object Collection>> that defines this class is that only instances of <<Class: Geometry, Geometry>> are allowed to be members of it and these are to be indicated with the `rdfs:member` property.
 
+[NOTE]
+====
+Some geometry literal formats, such as WKT, GML and GeoJSON, allow for collections of geometries. In WKT's case, this is indicated with the geometry type _GeometryCollection_. While the names of these objects are the same and this class' and the concepts are similar, there is no RDF/ontology relationship between literals of this type and this class. Since this class contains whole `geo:Geometry` instances, they may have much more information than geometry serialization within a literal geometry collection. 
```

Comment:
I think it is good to mention the other serialisations here as too: GML and KML have MultiGeometry, GeoJSON and WKT have GeomertryCollection.

Reply:
It seems to me that a main reason for collections of geometries existing in geometry serialisations is that it is a feature of  the SF specification. So maybe start the note with that?

Reply:
Thanks for point out these issues. Excellent eye for detail @FransKnibbe! I have updated the NOTE to reflect your suggestions.